scripts/release.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -e
+
+branch="$(git rev-parse --abbrev-ref HEAD)"
+
+if [[ $branch != "staging" || $branch != "staging-vue3" ]]; then
```

Comment:
hmm... this will always true... is this on purpose?

I mean.. `$branch` will always not be equal to one of the two values...
if branch is `staging` then it will not be `staging-vue3`so it is `true`, Do Im missing sth?

Reply:
Good catch, updated the if logic, thanks!